adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -423,5 +423,12 @@ class ADAMContextSuite extends ADAMFunSuite {
     val reads = sc.loadIndexedBam(path, refRegion)
     assert(reads.rdd.count == 1)
   }
+
+  sparkTest("loadIndexedVcf") {
+    val path = resourcePath("bqsr1.vcf")
+    val refRegion = ReferenceRegion("22", 0, 16050822)
```

Comment:
Can we use a more specific predicate than this? Ideally, we'd use one that illustrates that there is not a coordinate conversion issue.